static/app/components/tooltip.tsx

```diff
@@ -289,7 +290,6 @@ class Tooltip extends React.Component<Props, State> {
                 ref={arrowProps.ref}
                 data-placement={placement}
                 style={arrowProps.style}
-                background={(popperStyle as React.CSSProperties)?.background || '#000'}
```

Comment:
I didn't see anywhere that we were using `popperStyle.background` so we should be good here.